### DISCUSSION TITLE
SURF Innesco cannot have a specified flux or temperature and a MATL_ID (CHID: 240523_GEO)

### DISCUSSION THREAD

**Discussion by LordMicidial** [firemodels/fds#12994] [Q&A]
Dear all,

might you kindly help me on the error the compiler is giving to me? I can't understand where I'm wrong. These are the lines of the code under error:

&MATL ID                  ='Wood'
       DENSITY            = 180.845
       CONDUCTIVITY       = 0.27
       SPECIFIC_HEAT      = 1.21
       HEAT_OF_COMBUSTION = 17500/

&SURF ID='Innesco',
      RGB=255,102,0,
      HRRPUA=1000.0,
      RAMP_Q='Innesco_RAMP_Q',
      THICKNESS         = 0.01
      MATL_ID='Wood'/

&RAMP ID='Innesco_RAMP_Q', T=0.0, F=0.0, DEVC_ID='HRRTime'/
&RAMP ID='Innesco_RAMP_Q', T=50.0, F=1.0/
&RAMP ID='Innesco_RAMP_Q', T=100.0, F=4.0/
&RAMP ID='Innesco_RAMP_Q', T=150.0, F=1.0/
&RAMP ID='Innesco_RAMP_Q', T=200.0, F=17.0/
&RAMP ID='Innesco_RAMP_Q', T=250.0, F=27.0/
&RAMP ID='Innesco_RAMP_Q', T=300.0, F=38.0/
&RAMP ID='Innesco_RAMP_Q', T=350.0, F=52.0/
&RAMP ID='Innesco_RAMP_Q', T=400.0, F=68.0/
&RAMP ID='Innesco_RAMP_Q', T=450.0, F=86.0/
&RAMP ID='Innesco_RAMP_Q', T=4844.0, F=1.0/
&RAMP ID='Innesco_RAMP_Q', T=1.1921E+4, F=1.0/
&RAMP ID='Innesco_RAMP_Q', T=1.9371E+4, F=0.0/

Can you explain the meaning of the error? In my opinion, there is no temperature neither a flux specified within the SURF Code. 

As per screenshot attached, other two errors are mentioned related to the DEV and CTRLS I designed to control the HRR curve. Might the issue be related?

Also in this case, the code seems legit to me.

&PROP ID='Generic Commercial Link', QUANTITY='SPRINKLER LINK TEMPERATURE', ACTIVATION_TEMPERATURE=78.0/
&CTRL ID='ShutDown', FUNCTION_TYPE='TIME_DELAY', DELAY=10.0, LATCH=.FALSE., INPUT_ID='SLINK'/
&CTRL ID='SIMCTRL', LATCH=.FALSE., FUNCTION_TYPE='KILL', INPUT_ID='ShutDown'/
&DEVC ID='SLINK', PROP_ID='Generic Commercial Link', XYZ=116.5,72.5,6.2/
&DEVC ID='HRRTime', QUANTITY='TIME', XYZ=0.0,0.0,0.0, NO_UPDATE_DEVC_ID='SLINK'/

What do you think?

Thank you in advance for your time and consideration.

![image](https://github.com/firemodels/fds/assets/165648069/dc6e6689-b459-48e3-8c40-44442dc76536)

 ** EDIT -  I also have tried running the code eliminating  MATL_ID='Wood' from the SURF lines and i still get the same issue.

**Comment by mcgratta** [comment]
You are using an older version of FDS. Use the latest and tell me if the error persists.

**Reply by LordMicidial** [reply]
Many thanks. I though it could be a version issue as I recently changed it to an older one due to the Commercial GUI I just bought. I will refer to them even if I find way more easier to debug the code by hand.

Also, I'm seeing weirdly enormous numbers in the filename_HRR.csv and in the filename_devc.csv - is this related to potential error in the FDS version? Is it possible that FDS is computing with different units of measure or orders of magnitude than the standard ones given in the USER GUIDE? This seems very strange to me...

But please look at how it counts time and HRRTime in the filename_devc.csv
![image](https://github.com/firemodels/fds/assets/165648069/0b960cbb-e058-402b-8948-4a0ec73d3c9d)


Can you give me a clue as to what might be generating this? Also the filename_hrr.csv has absurdly large numbers.

**Comment by mcgratta** [comment]
You should always use the latest version of FDS. Your GUI should work with it because all that the GUI does is create the .fds file. If an input parameter has changed, you can simply edit the .fds file with the GUI or text editor. 

I have no idea what could be happening, and if this is an older version, I really don't care. Use the latest and we'll sort it out.

**Reply by LordMicidial** [reply]
I will, however this file was originally generated by Pyrosim, edited by me on the notepad and run on the cmdFDS compiler. I'll get through the modifications the GUI have designed by myself of course. Thanks!!
